prow/cluster/resources/gatekeeper-constraints/workloads/allowedImagesProwToolsNamespaces.yaml

```diff
@@ -0,0 +1,23 @@
+# Allow images needed to run supporting tools in Prow workload clusters.
+
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sAllowedImages
+metadata:
+  name: prow-tools-namespaces
+spec:
+  enforcementAction: warn
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Pod"]
+  parameters:
+    namespaces:
```

Comment:
namespaces can be defined in the `spec.match`, they don't need to even be in the constraint template

Reply:
Example https://github.com/kyma-project/test-infra/blob/main/prow/cluster/resources/gatekeeper-constraints/workloads/apparmor.yaml#L11

Reply:
corrected